src/watchdog/observers/inotify_c.py

```diff
@@ -195,7 +195,10 @@ def __init__(self, path, recursive=False, event_mask=WATCHDOG_ALL_EVENTS):
         self._path = path
         self._event_mask = event_mask
         self._is_recursive = recursive
-        self._add_dir_watch(path, recursive, event_mask)
```

Comment:
What is the purpose of this change?

Reply:
This change accidentally snuck in - it allows users to set watches on individual files. I'll open another PR for it so this PR stays clean.